apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -3035,6 +3035,80 @@ describe('Ticket', () => {
+
+                const [updatedTicket1] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.IN_PROGRESS } } })
+                expect(updatedTicket1).toHaveProperty('isCompletedAfterDeadline', false)
+
+                const [updatedTicket2] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.COMPLETED } } })
+                expect(updatedTicket2).toHaveProperty('isCompletedAfterDeadline', true)
+
+                const [updatedTicket3] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.CLOSED } } })
+                expect(updatedTicket3).toHaveProperty('isCompletedAfterDeadline', true)
+            })
+
+            test('Should be save false if the status change to "completed" and the deadline is not overdue', async () => {
+                const [ticket] = await createTestTicket(admin, organization, property, {
+                    deadline: dayjs().add(1, 'hour'),
+                })
+                expect(ticket).toHaveProperty('isCompletedAfterDeadline', false)
+                expect(ticket).toHaveProperty('status.id', STATUS_IDS.OPEN)
+
+                const [updatedTicket1] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.IN_PROGRESS } } })
+                expect(updatedTicket1).toHaveProperty('isCompletedAfterDeadline', false)
+
+                const [updatedTicket2] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.COMPLETED } } })
+                expect(updatedTicket2).toHaveProperty('isCompletedAfterDeadline', false)
+
+                const [updatedTicket3] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.CLOSED } } })
+                expect(updatedTicket3).toHaveProperty('isCompletedAfterDeadline', false)
+            })
+
+            test('Should be save true if the status change to "completed" and deadline is null', async () => {
+                const [ticket] = await createTestTicket(admin, organization, property)
+                expect(ticket).toHaveProperty('isCompletedAfterDeadline', false)
+                expect(ticket).toHaveProperty('status.id', STATUS_IDS.OPEN)
+
+                const [updatedTicket1] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.IN_PROGRESS } } })
+                expect(updatedTicket1).toHaveProperty('isCompletedAfterDeadline', false)
+
+                const [updatedTicket2] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.COMPLETED } } })
+                expect(updatedTicket2).toHaveProperty('isCompletedAfterDeadline', false)
+
+                const [updatedTicket3] = await updateTestTicket(admin, ticket.id, { status: { connect: { id: STATUS_IDS.CLOSED } } })
+                expect(updatedTicket3).toHaveProperty('isCompletedAfterDeadline', false)
+            })
+        })
```

Comment:
Add tests for cases:
1) Ticket returns to in progress from completed status
2) Ticket deadline updates and expired field changes to false